src/core/annotation.js

```diff
@@ -1377,6 +1382,7 @@ class Annotation {
 class AnnotationBorderStyle {
   constructor() {
     this.width = 1;
+    this.thickness = 1;
```

Comment:
Can we please call this `rawWidth` instead, to maintain consistency with the existing property (and with other code where we send not fully parsed/validated data)?